packages/expo-video/src/VideoView.types.ts

```diff
@@ -91,4 +91,30 @@ export interface VideoViewProps extends ViewProps {
    * @platform ios
    */
   contentPosition: { dx?: number; dy?: number } | undefined;
+
+  /**
+   * Determines the callback to call after the video enters Picture in Picture mode.
+   * @platform ios
+   */
+  onPictureInPictureStart?: () => void;
+
+  /**
+   * Determines the callback to call after the video exits Picture in Picture mode.
+   * @platform ios
+   */
+  onPictureInPictureStop?: () => void;
+
+  /**
+   * Determines whether the player should allow Picture in Picture.
+   * @default false
+   * @platform ios
+   */
+  allowsPictureInPicture?: boolean;
+
+  /**
+   * Determines whether the player should start Picture in Picture automatically when the app is minimized.
```

Comment:
```suggestion
   * Determines whether the player should start Picture in Picture (PiP) mode automatically when the app moves to the background.
```


We should probably note that there can be only one player in PiP mode 🤔